pkg/base/cockpit.js

```diff
@@ -2510,6 +2510,62 @@ function full_scope(cockpit, $) {
         return new HttpClient(endpoint, options || { });
     };
 
+    /* ---------------------------------------------------------------------
+     * Permission
+     */
+
+    var authority = null;
+
+    function Permission(options) {
+        var self = this;
+        self.allowed = null;
+
+        var user = cockpit.user;
+        var group = null;
+
+        if (options)
+            group = options.group;
+
+        function decide() {
+            var allowed = null;
```

Comment:
null and false have different meanings for return values from this function so the logic here wouldn't be correct. But if your point is to return early as flow control then we can make that change.